code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -39,67 +38,66 @@
 		S.take_overall_damage(anti_armour_damage*0.75, anti_armour_damage*0.25)
 	return BULLET_ACT_HIT
 
-/// Since some rockets have different booms depending if they hit a living target or not, this is easier than having explosive radius vars
+/// Since some missiles have different booms depending if they hit a living target or not, this is easier than having explosive radius vars
 /obj/projectile/bullet/a84mm/proc/do_boom(atom/target)
-	explosion(target, devastation_range = -1, heavy_impact_range = 1, light_impact_range = 3, flame_range = 4, flash_range = 1, adminlog = FALSE)
-
-/// PM9 standard rocket
-/obj/projectile/bullet/a84mm/he
-	name ="\improper HE missile"
-	desc = "Boom."
-	icon_state = "missile"
-	damage = 50
-	anti_armour_damage = 0
-
-/obj/projectile/bullet/a84mm/he/do_boom(atom/target, blocked=0)
 	if(!isliving(target)) //if the target isn't alive, so is a wall or something
 		explosion(target, heavy_impact_range = 1, light_impact_range = 2, flame_range = 3, flash_range = 4, explosion_cause = src)
 	else
 		explosion(target, light_impact_range = 2, flame_range = 3, flash_range = 4,  explosion_cause = src)
 
-/// PM9 weak rocket
+/// PM9 HEAP rocket - the anti-anything missile you always craved.
+/obj/projectile/bullet/a84mm/heap
+	name ="\improper HEAP rocket"
+	desc = "I am become death."
+	icon_state = "84mm-hedp"
+	damage = 80
+	armour_penetration = 100
+	dismemberment = 100
+	anti_armour_damage = 200
+
+/obj/projectile/bullet/a84mm/heap/do_boom(atom/target, blocked=0)
+	explosion(target, devastation_range = -1, heavy_impact_range = 1, light_impact_range = 3, flame_range = 4, flash_range = 1, adminlog = FALSE)
+
+/// PM9 weak rocket - just kind of a failure
 /obj/projectile/bullet/a84mm/weak
-	name ="low-yield HE missile"
+	name ="low-yield rocket"
 	desc = "Boom, but less so."
 	damage = 30
-	anti_armour_damage = 0
 
 /obj/projectile/bullet/a84mm/weak/do_boom(atom/target, blocked=0)
 	if(!isliving(target)) //if the target isn't alive, so is a wall or something
 		explosion(target, heavy_impact_range = 1, light_impact_range = 2, flame_range = 3, flash_range = 4, explosion_cause = src)
 	else
 		explosion(target, light_impact_range = 2, flame_range = 3, flash_range = 4, explosion_cause = src)
 
-/// Mech BRM-6 missile
-/obj/projectile/bullet/a84mm_br
-	name ="\improper HE missile"
-	desc = "Boom."
-	icon_state = "missile"
+/// SRM-8 Missile - Used by the SRM-8 Exosuit missile rack.
+// Employed by Nuclear Operatives Maulers and Nanotrasen Marauders and Seraphs to kill everything and anyone.
```

Comment:
im not sure using autodoc like that works, can you just do multiline version of autodoc?

Reply:
Ye